pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -100,14 +99,6 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 		return err
 	}
 
-	if err := r.ensureTargetNamespaceExists(ctx, tc); err != nil {
-		return err
-	}
-
-	if err := common.CreateOperatorVersionConfigMap(r.manifest, tc); err != nil {
-		return err
-	}
-
```

Comment:
@PuneetPunamiya i think we should retain all the other code except this one. 

i think, creation of targetnamespace from tetkonConfig reconcier is a great idea as it gives us better control over the ownership of target namespace.

Reply:
in my opinion, this pr should remove only
```golang
if err := common.CreateOperatorVersionConfigMap(r.manifest, tc); err != nil {
		return err
	}
```

cc @sm43

Reply:
@nikhil-thomas 
Yep, creating namespace in TektonConfig might be a good idea if we can add tektonconfig as owner to namespace.

Reply:
yeah make sense 👍🏻 
I've updated the patch